src/dataframe/dataframe.jl

```diff
@@ -98,10 +98,32 @@ type DataFrame <: AbstractDataFrame
     end
 end
 
+# Return and AbstractVector of length `len` filled with `x`.
+# `x` is recycled if `len(x)` is an even multiple of `len`.
```

Comment:
This is `length(x)`. Also, the description doesn't sound completely accurate: `x` is always recycled an integer number of times, so that the length is less than or equal to `len`. Though the behaviour you describe would be sensible (i.e. raise an error if not a multiple).